ComprefaceTestApp/DTOs/ExampleSubject/AddExampleSubject/AddExampleSubjectRequest.cs

```diff
@@ -1,14 +1,10 @@
-﻿using System.Text.Json.Serialization;
+﻿namespace ComprefaceTestApp.DTOs.ExampleSubject.AddExampleSubject;
 
-namespace ComprefaceTestApp.DTOs.ExampleSubject.AddExampleSubject;
-
-public class AddExampleSubjectRequest
+public class AddExampleSubjectRequest : SubjectBase
 {
-    public string Subject { get; set; }
-    
     public decimal? DetProbThreShold { get; set; }
 
-    public string FilePath { get; set; }
+    public string? FilePath { get; set; }
 
-    public string FileName { get; set; }
+    public string? FileName { get; set; }
 }
```

Comment:
Same here

Reply:
From C# 8.0 null-context definition has appeared. So that's why we need mark string like a nullable. Also we can set up it global in the project and disable this warnings.